### DISCUSSION TITLE
请问v2的夜间模式开关在哪里

### DISCUSSION THREAD

**Discussion by sino1641** [the1812/Bilibili-Evolved#2503] [提问 / Q&A] [answered]
RT，请问v2的夜间模式开关在哪里？
恕我眼拙看不到开关，设置里搜索`夜间|暗黑|dark mode`均无结果。
尝试在仓库里装顶栏的暗黑开关，但是刷新页面后也找不到。。

**Comment by Geeyun-JY3** [comment, marked as answer]
你装了 `夜间模式` 这个组件没有：https://github.com/the1812/Bilibili-Evolved/blob/v2/doc/features/features.md#夜间模式
![image](https://user-images.githubusercontent.com/49396421/141397143-0657dfd8-6638-4d63-bf09-77349da67e6d.png)

**Reply by sino1641** [reply]
> 你装了 `夜间模式` 这个组件没有：[https://github.com/the1812/Bilibili-Evolved/blob/v2/doc/features/features.md#夜间模式](https://github.com/the1812/Bilibili-Evolved/blob/v2/doc/features/features.md#%E5%A4%9C%E9%97%B4%E6%A8%A1%E5%BC%8F) 

了解~
我仓库里也搜不到
直接进行一个重装就搜到了
谢谢回复
